src/simulator.ts

```diff
@@ -145,6 +142,7 @@ export class Simulator {
           deviceConnectionStrings,
         );
       }
+      vscode.commands.executeCommand("azure-iot-toolkit.refresh");
```

Comment:
What's the purpose of refresh here?

Reply:
When user choose to launch the webview with no connection string, it will call a method in Utility to ask user to set a string first, but that function just stores the string and will not load the devices in the explorer. I call the refresh here so that the explorer can load the devices, and thus it follows that user can right-click at a device and start the webview with that device.